test/data_static.jl

```diff
@@ -72,3 +72,79 @@ test_toyModel() = CoreModel(
     ["r1", "m1t", "m3t", "EX_m1(e)", "EX_m3(e)", "EX_biomass(e)", "biomass1"],
     ["m1[c]", "m3[c]", "m2[c]", "m1[e]", "m3[e]", "biomass[c]"],
 )
+
+const reaction_standard_gibbs_free_energies = Dict(
```

Comment:
how was this data actually generated?

Reply:
cc @stelmo 

I thought I asked for this in review as well, AFAIK it's described in tutorials (where the same data is used) but yeah a link would help here a lot.

Reply:
cool, will add one